pkgs/development/python-modules/seahub/default.nix

```diff
@@ -1,5 +1,8 @@
-{ lib, fetchFromGitHub, python3Packages, makeWrapper }:
+{ lib, fetchFromGitHub, python3Packages, makeWrapper, fetchPypi }:
 
+let
+  django-webpack-loader = import ./django-webpack-loader.nix { inherit lib fetchPypi; buildPythonPackage = python3Packages.buildPythonPackage; };
+in
```

Comment:
We can't do this in pythonPackages. This need to be done outside in the package itself.

Reply:
I was afraid this would be the case :/
What's the best approach: defining a django-webpack-loader_0_7_0 package?

Reply:
Maybe like this: https://github.com/NixOS/nixpkgs/blob/894fb34b239d37acbd670f0322a6739a8d668916/pkgs/top-level/python-packages.nix#L7889

I feel like adding a `django-webpack-loader_0_7_0` package is unnecessary and cluttering `pythonPackages` if its only used for seahub.

Reply:
> What's the best approach: defining a django-webpack-loader_0_7_0 package?

That is also not allowed in pythonPackages. We would need to do https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#overriding-python-packages-overriding-python-packages in the final package.

Reply:
Thank you both for your feedback, I've updated the PR to override the input django-webpack-loader.